units/src/amount/tests.rs

```diff
@@ -568,44 +560,45 @@ fn from_str() {
 fn to_from_string_in() {
     use super::Denomination as D;
     let ua_str = Amount::from_str_in;
-    let ua_sat = Amount::from_sat;
+    let ua_sat = Amount::from_sat_unchecked;
     let sa_str = SignedAmount::from_str_in;
+    let sa_sat = SignedAmount::from_sat_unchecked;
 
-    assert_eq!("0.5", Amount::from_sat(50).to_string_in(D::Bit));
-    assert_eq!("-0.5", SignedAmount::from_sat(-50).to_string_in(D::Bit));
-    assert_eq!("0.00253583", Amount::from_sat(253_583).to_string_in(D::Bitcoin));
-    assert_eq!("-5", SignedAmount::from_sat(-5).to_string_in(D::Satoshi));
-    assert_eq!("0.1", Amount::from_sat(100_000_00).to_string_in(D::Bitcoin));
-    assert_eq!("-0.1", SignedAmount::from_sat(-100_000_00).to_string_in(D::Bitcoin));
+    assert_eq!("0.5", sa_sat(50).to_string_in(D::Bit));
```

Comment:
```suggestion
    assert_eq!("0.5", ua_sat(50).to_string_in(D::Bit));
```
`Amount::from_sat` should be changed to `ua_sat`

Reply:
oooph, face palm. Good attention to detail, thanks man. I split the offending patch up into three


dc8ad85e Add local variable to use as constructor
4bc65d90 Use ua_sat throughout test function
93b1ce3b Introduce Amount::from_sat_unchecked